package.json

```
@@ -6,6 +6,7 @@
     "bench": "vitest bench",
     "build": "npx lerna run build --stream",
     "clean": "npx lerna clean",
```

nit: these `npx` prefixes are unnecessary in npm scripts:

```suggestion
    "build": "lerna run build --stream",
    "clean": "lerna clean",
```